v0/pipeline.json

```diff
@@ -31763,6 +31773,16 @@
                 "connectorRef" : {
                   "type" : "string"
                 },
+                "globalSettings" : {
+                  "oneOf" : [ {
+                    "type" : "object",
+                    "additionalProperties" : {
```

Comment:
oneOf condition is to allow configuration at runtime?

Reply:
yes

Reply:
@Abhishek-CDC changed it to not allow runtime at complete field, which we are not providing in liquibase command step as well